CodeGeneration/Sources/SyntaxSupport/AvailabilityNodes.swift

```diff
@@ -118,42 +118,54 @@ public let AVAILABILITY_NODES: [Node] = [
     ]
   ),
 
-  // version-tuple -> integer-literal
-  //                | float-literal
-  //                | float-literal '.' integer-literal
+  // version-tuple-element -> '.' float-literal |
+  //                          '.' interger-literal
```

Comment:
This is completely out of date since we no longer parse float literals into a version tuple. I think you can just remove it.